bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md

```diff
@@ -64,3 +64,16 @@
 2. Check if auth credentials are correct.
 3. Check if server is running.
 
+## Limitations and issues
+
+There are known limitations and issues that are tracked by OpenSearch including the items listed below.
+
+| Issue | Description |
+| ---- | ----------- |
+| Visualizations without numerical columns do not work | Visualizations will work if there is a numerical column included due to a known limitation in OpenSearch regarding subqueries. However, visualizations with columns only of type string works as well. |

```

Comment:
I think it is assumed
>  Visualizations will **not** work if there is a numerical column included ...
> ... only of type string **work** as well.
in the second column

Reply:
Do we need to reference here the bug? https://github.com/opensearch-project/sql/issues/347